io/net/sriov_device_test.py

```diff
@@ -249,13 +253,13 @@ def test_remove_logical_device(self):
         if self.max_sriov_port:
             self.cancel("Test unsupported")
 
-        for mac, slot in zip(self.mac_id, self.sriov_adapter):
-            logical_port_id = self.get_logical_port_id(mac)
+        for mac, slot, sriov_roce in zip(self.mac_id, self.sriov_adapter, self.sriov_roce):
+            logical_port_id = self.get_logical_port_id(mac, sriov_roce)
```

Comment:
what if the `sriov_roc` parameter is not there in yaml?
what value `self.sriov_roce` will hold here?